pymongo/encryption_options.py

```diff
@@ -59,6 +59,17 @@ def __init__(self, kms_providers, key_vault_namespace,
               - `aws`: Map with "accessKeyId" and "secretAccessKey" as strings.
                 These are the AWS access key ID and AWS secret access key used
                 to generate KMS messages.
+              - `azure`: Map with "tenantId", "clientId", and "clientSecret" as
+                strings. Additionally, "identityPlatformEndpoint" may also be
+                specified as a string (defaults to 'login.microsoftonline.com').
+                These are the Azure Active Directory credentials used to
+                generate Azure Key Vault messages.
+              - `gcp`: Map with "email" as a string and "privateKey" as a
+                base64 encoded string or `bytes`. Python 2 users must specify
+                "privateKey" as a base64 encoded string. Additionally,
```

Comment:
> Python 2 users must specify "privateKey" as a base64 encoded string

This isn't true at least not anymore. I suggest:
```
"privateKey" as `bytes` or a base64 encoded string (unicode on Python 2).
```

Reply:
Done.